benefits/urls.py

```diff
@@ -27,6 +27,7 @@
     path("enrollment/", include("benefits.enrollment.urls")),
     path("i18n/", include("django.conf.urls.i18n")),
     path("oauth/", include("benefits.oauth.urls")),
+    path("oauth/", include("cdt_identity.urls")),
```

Comment:
I don't think we want this (yet) -- right?

Reply:
I thought so too, that we wouldn't need it but I came across the exception described in this [comment](https://github.com/cal-itp/benefits/pull/2793#discussion_r2024897356)